src/main/java/teammates/common/datatransfer/FeedbackParticipantType.java

```diff
@@ -141,4 +141,25 @@ public String getDisplayNameGiver() {
     public String getDisplayNameRecipient() {
         return displayNameRecipient;
     }
+    
+    /**
+     * @return A user-friendly {@FeedbackParticipantType} list representing participant.
+     */
+    public static List<FeedbackParticipantType> getParticipantListFromCommaSeparatedValues(
+            String commaSeparatedValues) {
+        List<FeedbackParticipantType> participantList = new ArrayList<FeedbackParticipantType>();
+
+        if (commaSeparatedValues == null
+                || commaSeparatedValues.isEmpty()) {
+            return participantList;
+        }
+
+        String[] splitString = commaSeparatedValues.split(",");
```

Comment:
Since this is only used once in the for loop, you can do this instead (in the for loop):
```java
for (String value : commaSeparatedValues.split(",")) {
```